app/decorators/case/sar_decorator.rb

```diff
@@ -1,2 +1,15 @@
 class Case::SARDecorator < Case::BaseDecorator
+
+  def subject_type
+    object.subject_type.humanize
+  end
+
+
+  def third_party
+    object.third_party ? 'Yes' : 'No'
+  end
+
+  def sar_response_address
+    object.email.present? ? object.email : object.postal_address
```

Comment:
You can use the `reply_method` enum's values to check whether the submitter checked `send_by_post` or `send_by_email`:

```
    object.send_by_email? ? object.email : object.postal_address
```

Reply:
The reason we did it that way is that Mara said that we could very well expect to have both email and a postal addresses on a case.